src/app/pages/profile/profile/profile.component.ts

```diff
@@ -26,7 +26,7 @@ import {combineLatest, of} from 'rxjs';
 })
 export class ProfileComponent extends PageComponent {
 
-    static craftingJobs = [
+    private craftingJobs: any[] = [
```

Comment:
iirc, this was set to static because it's used somewhere else, have you checked that is wasn't?

Reply:
Just checked, it's not used anywhere else. nvm